components/supervisor/pkg/serverapi/publicapi.go

```diff
@@ -52,6 +52,7 @@ type ServiceConfig struct {
 	WorkspaceID       string
 	OwnerID           string
 	SupervisorVersion string
+	EnabledConfigCat  bool
```

Comment:
```suggestion
	ConfigcatEnabled  bool
```
for more standard naming